frontends/open-discussions/src/pages/CourseIndexPage.js

```diff
@@ -87,6 +86,18 @@ export default function CourseIndexPage({ history }: Props) {
 
   useLearningResourcePermalink()
 
+  const [searchText, setSearchText] = useState("")
```

Comment:
Previously the `<Searchbox />` component in ol-search-ui supported both "controlled" and "uncontrolled" usage (See https://reactjs.org/docs/uncontrolled-components.html). 

When I separated Searchbox and SearchInput, I dropped the support for uncontrolled usage. This simplifies stuff and also fixes the bug that @abeglova noticed at https://discussions-rc.odl.mit.edu/learn where previously if you submitted search through the "Explore" button, the search param was not forwarded to search page. 

Hopefully nothing else will touch open-discussions for a while...